tap_codat/streams.py

```diff
@@ -26,6 +26,18 @@ def write_records(self, records):
         singer.write_records(self.tap_stream_id, records)
         self.metrics(records)
 
+    def log_additional_properties(self, ctx, records):
```

Comment:
Am I correct in my understanding that this is using the singer transform code to output an error so we can just log it out for visibility? If so, can we add a comment here to make that more clear?